src/dodal/devices/eiger.py

```diff
@@ -56,17 +57,25 @@ def set(self, value, *, timeout=None, settle_time=None, **kwargs):
 
     stale_params = Component(EpicsSignalRO, "CAM:StaleParameters_RBV")
     bit_depth = Component(EpicsSignalRO, "CAM:BitDepthImage_RBV")
-
     filewriters_finished: StatusBase
 
     detector_params: DetectorParams | None = None
 
     arming_status = Status()
     arming_status.set_finished()
 
-    def __init__(self, beamline: str = "i03", *args, **kwargs):
+    def __init__(self, beamline: str = "i03", detector_id: int = 78, *args, **kwargs):
```

Comment:
Nit: There's not really a reason that 78 should be the default, that's just what is on i03. I don't think there is a sensible default.